pkg/exec/exec.go

```diff
@@ -0,0 +1,40 @@
+package exec
```

Comment:
Why is this its own package? It seems like this would be fine living in the util package, where we wouldn't overlap with the core os/exec namespace. I know we can define alternative names for packages, but it seems like it would be preferable to avoid having to rename core packages, and avoid possible confusion.